README.adoc

```diff
@@ -228,6 +172,69 @@ The reply should be as follows:
 ----
 ====
 
+
+
+== Preparing to Build the Application
+
+To package and run the application, we need to provide an external MySQL database rather than using Spring Boot Docker Compose Support.
+For this task, we can reuse the `compose.yaml` provided with a few modifications.
```

Comment:
Change "`compose.yaml` provided" to "provided `compose.yaml` file"

Change the period to a colon.